### PR TITLE
[AC-2727] feat: widget modularization - scroll to bottom button

### PR DESCRIPTION
**Pull request by bang9**
I divide the detailed tasks of modularization and work on them separately.

## Changes

- Added ScrollToBottomButton component.
- Update structure of foundation components

ticket: [AC-2727]

[AC-2727]: https://sendbird.atlassian.net/browse/AC-2727?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/foundation/components/Label/css.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+//     ${styles}
+//   }
+// `;
+//
+// const sendbirdLabelMobile = css`
+//   ${mobileMixin(`
+//     -webkit-user-select: none;
+//     -webkit-touch-callout: none;
+//   `)};
+// `;
```
Comment (chohongm) [inline src/foundation/components/Label/css.ts:14]: Should we keep these comments?

File: src/foundation/components/Label/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      | 'subtitle1'
+      | 'subtitle2'
+      | 'body1'
+      | 'body2'
+      | 'button1'
+      | 'button2'
+      | 'button3'
+      | 'caption1'
+      | 'caption2'
+      | 'caption3';
```
Comment (chohongm) [inline src/foundation/components/Label/index.tsx:33]: will these be resused? maybe define `ElementType`?

File: src/foundation/components/Label/index.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+type AsProp<C extends ElementType> = {
+  as?: C;
+};
+type PropsToOmit<C extends ElementType, P> = keyof (P & AsProp<C>);
+type PolymorphicComponentProps<C extends ElementType, Props = NonNullable<unknown>> = Props &
+  AsProp<C> &
+  Omit<React.ComponentPropsWithoutRef<C>, PropsToOmit<C, Props>>;
+
+type HTMLProps<C extends ElementType> = PolymorphicComponentProps<C, HTMLAttributes<HTMLElement>>;
```
Comment (chohongm) [inline src/foundation/components/Label/index.tsx:17]: here and above is a bit complex to understand. Let me ask you later.

**Review by chohongm — APPROVED** [review]
LGTM

File: src/foundation/components/Label/css.ts (hunk 1 above)
Comment (bang9) [reply src/foundation/components/Label/css.ts:14]: I've left only the code for the part that needs to be implemented additionally.

File: src/foundation/components/Label/index.tsx (hunk 3 above)
Comment (bang9) [reply src/foundation/components/Label/index.tsx:17]: This is simply for accurate type inference. To fully understand it, you need to study the inference in TypeScript further. If you do not wish to deep dive into it, you can just understand it as a part for type inference.

File: src/foundation/components/Label/index.tsx (hunk 2 above)
Comment (bang9) [reply src/foundation/components/Label/index.tsx:33]: I didn't quite understand what you are asking.

**Pull request merged**